demo/js/demo.js

```diff
@@ -452,25 +452,6 @@ window.onload = function() {
                     n = behavior.getColumn(idx.TOTAL_NUMBER_OF_PETS_OWNED).getValue(y);
                     hex = (155 + 10 * (n % 11)).toString(16);
                     config.backgroundColor = '#' + hex + hex + hex;
-                } else {
-                    switch (config.y % 6) {
```

Comment:
While we're at it. We can simplify getCell even further. Removing https://github.com/openfin/fin-hypergrid/pull/389/files#diff-c4ec3078804b5269f20ac55a6f14d676R443